package.json

```diff
@@ -11,10 +12,10 @@
     "crawlers",
     "webcrawler"
   ],
-  "homepage": "http://github.com/seantomburke/sitemapper",
```

Comment:
This PR changes the main URLs of the underlying package, so I wouldn't be able to merge this in. The NPM package needs to reference this repository and not the fork.